tests/CMakeLists.txt

```diff
@@ -20,9 +20,36 @@ if(NOT BYO_CRYPTO)
     add_net_test_case(TLSContextResourceSafety)
     add_net_test_case(TLSContextUninitializedNewConnectionOptions)
     add_test_case(Sigv4aSigningTestCredentials)
+
     add_net_test_case(IoTMqtt311ConnectWithNoSigningCustomAuth)
     add_net_test_case(IoTMqtt311ConnectWithSigningCustomAuth)
+    add_net_test_case(IoTMqtt311ConnectWithSigningCustomAuthWebsockets)
+    add_net_test_case(IoTMqtt311ConnectWithPKCS11)
     add_net_test_case(IoTMqtt311ConnectWithPKCS12)
+    add_net_test_case(IoTMqtt311ConnectWithWindowsCert)
+    add_net_test_case(IoTMqtt311ConnectWSDefault)
+    add_net_test_case(IoTMqtt311ConnectWSStatic)
+    add_net_test_case(IoTMqtt311ConnectWSCognito)
+    add_net_test_case(IoTMqtt311ConnectWSProfile)
+    add_net_test_case(IoTMqtt311ConnectWSEnvironment)
+
+    if(ENABLE_MQTT5_TEST)
```

Comment:
I think we can remove this bool from here since net tests skip automatically if we're not connected to a network and the individual tests skip if it's missing an env variable they need? If we do remove this, we can also remove it from the CMakeList file that sets it (which may mean it's just always on unless something changes it elsewhere)